src/features/events/components/EventOverviewCard/index.tsx

```diff
@@ -42,16 +56,40 @@ const EventOverviewCard: FC<EventOverviewCardProps> = ({
   const [locationId, setLocationId] = useState(
     eventData?.location.id ?? undefined
   );
+
+  const [startDate, setStartDate] = useState<Dayjs | undefined>(
+    eventData?.start_time
+      ? dayjs(removeOffset(eventData.start_time))
+      : undefined
+  );
+  const [endDate, setEndDate] = useState<Dayjs | undefined>(
+    eventData?.end_time ? dayjs(removeOffset(eventData.end_time)) : undefined
+  );
+  const [openButton, setOpenButton] = useState(false);
```

Comment:
Am I right in my understanding of the code that this is the state that controls whether there should be an end date input visible? If that's the case, I think the name for this state is not communicating very well what it's fore. It took me a while to figure it out. Maybe call it `showEndDate` or `endDateVisible` or something like that?

Reply:
You´re right! I'll change it!